components/dropdown/Dropdown.js

```diff
@@ -180,7 +180,8 @@ const factory = (Input) => {
         <li
           key={idx}
           className={className}
-          onMouseDown={!item.disabled && this.handleSelect.bind(this, item[valueKey])}
+          onMouseDown={!item.disabled}
+          onClick={this.handleSelect.bind(this, item[valueKey])}
```

Comment:
I have the feeling that `onMouseDown` is doing nothing since it's expecting a function, and `onClick` is handling the click event even if the dropdown is disabled.

Since you can't open open the dropdown if it's disabled it makes no sense to have that check in the values of the dropdown, can you remove it?

Can you check if it works as expected with the dropdown disabled?